charts/fybrik/values.yaml

```diff
@@ -160,19 +178,10 @@ manager:
 
   podAnnotations: {}
 
-  podSecurityContext:
-    runAsNonRoot: true
+  podSecurityContext: {}
```

Comment:
please add a comment that this setting overwrites the global podSecurityContext settings

Reply:
Fixed. Thanks